skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,294 @@
+import numpy as np
+import scipy as sp
+
+
+class TpsTransform:
```

Comment:
I am growing more doubtful if the class based approach does actually give us an advantage beyond being consistent with the geometric transforms. `TpsTransform` is basically just a fancy wrapper around `parameters` ($f(x,y) = L^-1 Y$) and `control_points` (not sure about the corresponding symbol in the paper yet).

I am currently asking myself, is it possible at any stage of the computation to combine TpsTransforms A and B such that the resulting TpsTransform C is the same as A and B applied successively.. :thinking:

Reply:
You are correct about it being a wrapper. The `control_points` is just a placeholder for `from_points`. Maybe my choice of variable name creates some confusion. 
You have raised an Interesting about the possibility of combining different transform objects. Would be worth investigating.